configure.ac

```diff
@@ -213,6 +213,12 @@ AC_ARG_ENABLE(
     [enable_ml_tests="yes"],
     [enable_ml_tests="no"]
 )
+AC_ARG_ENABLE(
+    [aclk_ssl_debug],
+    [AS_HELP_STRING([--enable-aclk-ssl-debug], [Enables possibility for SSL key logging @<:@default no@:>@])],
```

Comment:
Netdata installer doesnt support this option only configure. Reason is I do not want this feature be accessible to regular users. Developers who need it will not have issue installing with autoreconf, configure, make.